src/components/Modal/Modal.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import { animated, useTransition } from 'react-spring'
+import styled from 'styled-components'
+import Button from '../Button/Button'
+
+Modal.propTypes = {
+  modalText: PropTypes.string,
+  modalHeader: PropTypes.string,
+  modalVisible: PropTypes.bool,
+  setModalVisible: PropTypes.func,
+  onCloseModalGoToPath: PropTypes.string,
+}
+
+export default function Modal({
+  modalText,
```

Comment:
Da du hier im Kontext Modal bist, kannst du glaube ich, den ersten Teil "modal" immer weglassen, zumindest bei Text und Header.

Reply:
Stimmt! Man sieht ja, dass es der Text in der Modal Komponenten ist :)